src/RcppExports.cpp

```diff
@@ -7,6 +7,11 @@
 
 using namespace Rcpp;
 
+#ifdef RCPP_USE_GLOBAL_ROSTREAM
+Rcpp::Rostream<true>&  Rcpp::Rcout = Rcpp::Rcpp_cout_get();
+Rcpp::Rostream<false>& Rcpp::Rcerr = Rcpp::Rcpp_cerr_get();
+#endif
+
```

Comment:
Hi @eddelbuettel. Sorry for not checking RcppExports.cpp for that particular commit (remember that this was a thing last time). 

It think it appeared because of my use of `Rcpp::Rcout` to do some debugging thing and then I forgot to remove print statements before committing (as I usually do) to have a clean commit.

I think it probably appears via compile attributes
- `R CMD INSTALL --preclean --no-multiarch --with-keep.source` is executed directly in vscode whenever I build rcppsmc there ->lines 10-13 do NOT appear
- "Clean and rebuild in RStudio" from reading the RStudio pane: 
   ```
   ==> Rcpp::compileAttributes()

   * Updated src/RcppExports.cpp
   * Updated R/RcppExports.R

   ==> R CMD INSTALL --no-multiarch --with-keep.source rcppsmc
   ```

So the latter specifically adds those lines while my vscode build execution does not (compile attributes is missing).

Now here is the thing (why this might be worth mentioning):
1. Now if comment out that particular statement like so `// Rcpp::Rcout <<....` and rebuild neither RStudio nor vscode removes that entry in RcppExports.cpp.
2. If I delete that (commented out) line alltogether neither RStudio nor vscode remove the entry in RcppExports.cpp

Reply:
(That is a non issue and simply due to Rcpp 1.0.7 being newer than 1.0.6 or other. It's a minor new feature for the output device Rcpp uses -- see the release notes for Rcpp 1.0.7.  In short nothing to worrry about.   [ Unless I completely misunderstand what you are asking. ] )